code/api/src/modules/user/types.js

```diff
@@ -1,42 +1,42 @@
 // Imports
-import { GraphQLObjectType, GraphQLString, GraphQLInt } from 'graphql'
```

Comment:
data types to allow for access of various types of data

Reply:
Cool. I don't think I fully cognized that you need to import each individual type. Good to know!

Reply:
Looks good.